t/12-dbic-ResultSet-Run.t

```diff
@@ -8,38 +8,58 @@ use t::dbic_util;
 my $schema = t::dbic_util->new->test_schema();
 
 subtest 'find a run using given attributes' => sub {
-  plan tests => 9;
+  plan tests => 11;
 
   my $rs = $schema->resultset('Run');
-  ok ($rs->can('find_with_attributes'), 'method find_with_attributes is implemented');
+  ok ($rs->can('find_with_attributes'),
+    'method find_with_attributes is implemented');
 
   my @args = qw/20250127_AV244103_NT1850075L 2427452508 AV2/;
   while (@args) {
     pop @args;
-    throws_ok { $rs->find_with_attributes(@args) }
-      qr/One of runfolder name, flowcell ID or barcode or instrument name is undefined/,
+    throws_ok { $rs->find_with_attributes('2427452508') }
```

Comment:
This appears to be iterating three times with the same test. I suggest abandoning `@args` here. It seems to be redundant.